cypress/integration/demo-page.spec.ts

```diff
@@ -16,7 +16,8 @@ import {
   getDeleteButton,
   getSelectAllButton,
   shouldBeInSelectMode,
-  getSelectCount
+  getSelectCount,
+  disableToggleWithoutShortcut
```

Comment:
Would prolly rename this to the name I have suggested above, unless you have another name that we could discuss.